readme.md

```diff
@@ -0,0 +1,104 @@
+# Tech Spec - Teleport Challenge
+
+## Overview
+To be implemented:
+
+1. `Job` library for starting, stopping, and getting status and streamed logs of jobs.
+2. API that implements the `Job` library, permitting clients to call the 4 public methods.
+3. CLI Client that calls API, using command line args to specify API methods.
+
+## Job Library
+`Job` type will include public fields:
+
+- Command
+- Args
+- ProcessID
+
+##### Implement methods for:
+
+- Start
+	- Assure job command exists, e.g. `which <cmd>`.
```

Comment:
how will you do this programmatically?

Reply:
I added a line about `exec.LookPath`. As always, I'm open to alternate suggestions.